src/sentry/receivers/features.py

```diff
@@ -156,12 +156,10 @@ def record_project_created(project, user, **kwargs):
 
 @member_joined.connect(weak=False)
 def record_member_joined(member, organization_id: int, **kwargs):
-    if FeatureAdoption.objects.record(
+    FeatureAdoption.objects.record(
```

Comment:
What's wrong about these conditionals? The `record()` method returns bool.

Reply:
I added these, like an idiot, they are not intended.